kolibri/core/assets/src/views/core-base.vue

```diff
@@ -143,7 +144,5 @@
     overflow-x: hidden
     right: 0
     bottom: 0
-    padding-bottom: 40px
-    padding: 32px
```

Comment:
let's leave the bottom padding fixed at 40px, and have this change only affect left- and right-padding. Keeping bottom padding large can be helpful for reading content so you can scroll down a little further.